package.json

```diff
@@ -43,7 +43,6 @@
     "jest": "^29.7.0",
     "npm-run-all2": "^5.0.2",
     "prettier": "^3.2.4",
-    "ts-jest": "^29.1.2",
```

Comment:
we use `@swc/jest` instead, not needed

Reply:
looks like its used here? https://github.com/interledger/rafiki/blob/main/jest.config.js#L4

Reply:
@BlairCurrey was about to merge until I saw this, fixed now :+1: